.github/workflows/lint_and_test.yml

```diff
@@ -0,0 +1,60 @@
+name: Lint & Test
+
+on: push
+
+jobs:
+  lint:
+    name: Lint
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install system dependencies
+        run: sudo apt-get install -y libsqlite3-dev
+      - uses: actions/checkout@master
+      - name: Setup ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: 2.7.8
+      - name: Install bundle
+        run: bundle
+      - name: Run rubocop
+        run: bundle exec rubocop
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
```

Comment:
@lmaiwald I just noticed, there's kt-paperclip 7.2 which should probably be tested (I'd suggest removing pc 7.0). 

Also, do we want to test ruby 3.2 and rails 7? If yes, do we also want to remove pc 7.1 in test matrix because matrix is too big otherwise? WDYT?